makePoplogCommander.sh

```diff
@@ -12,7 +12,15 @@ set -euo pipefail
 # DEFAULT_RUN_VARIANT is the default for scripts.
 ################################################################################
 
-VARIANTS=(nox xt xm)
+# The options are what the user types. They have to be translated into builds.
+# e.g. --gui=xt has to become xt. The reason for that is that the 'editions'
+# of Poplog relate to the mutually exclusive build options. The mapping is 
+# implicitly defined by matching position in the two (congruent) arrays.
+VARIANT_OPTIONS=(--no-gui --gui=xt --gui=motif)
+VARIANT_BUILDS=(nox xt xm)
+
```

Comment:
Maybe we should add a little assertion here to make sure these are the same length as a sanity check?
e.g. `[[ ${#VARIANT_OPTIONS[@]} -eq ${#VARIANT_BUILDS[@]} ]] || { echo "VARIANT_OPTIONS should have the same number of items as VARIANT_BUILDS but did not"; exit 1; }`

Reply:
Although it is overkill, it serves as a way of saying 'these two must be the same length" so I have just adopted it. Thanks.